numpy/add_newdocs.py

```diff
@@ -3070,6 +3070,14 @@ def luf(lamdaexpr, *args, **kwargs):
     30
     >>> np.prod(x.shape)
     30
+    
+    Notes
+    -----
+    `a.size` returns a standard arbitrary precision Python integer. This 
+    may not be the case with other methods of obtaining the same value
+    (like the suggested ``np.prod(a.shape)``, which returns an instance
+    of ``np.int_``), and may be relevant if the value is used further in
+    calculations that may overflow a fixed size integer type.
```

Comment:
`Notes` seems to always come before `Examples` in all other docstrings. Could you please swap the order to keep consistency?

Reply:
Fixed